### ISSUE TITLE
searchGuildMembers endpoint erroring

### ISSUE THREAD

**Issue by One-Nub** [l7ssha/nyxx#102]
**Describe the bug**
While trying to search members in a guild (utilizing `ctx.guild.searchMembers()`) the result will return with an error + traceback.
```
[2020-12-22 15:17:50.679407] [SEVERE] [Client] ERROR: [Code: 400] [Message: {"code": 50035, "errors": {"query": {"_errors": [{"code": "BASE_TYPE_REQUIRED", "message": "This field is required"}]}}, "message": "Invalid Form Body"}] 
 Instance of '_StringStackTrace'
[2020-12-22 15:17:50.679972] [SEVERE] [Client] ERROR: type 'HttpResponseError' is not a subtype of type 'HttpResponseSuccess' in type cast 
 #0      _HttpEndpoints.searchGuildMembers (package:nyxx/src/internal/_HttpEndpoints.dart:582:61)
```

**To Reproduce**
1. Attempt to find a user in a guild from a command via `ctx.guild!.searchMembers(query)`
2. Observe

**Expected behavior**
Preferably not an error, and a found Stream of Member(s) will be returned

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.10
 - Dart Version: 2.12.0-133.2.beta
 - Nyxx Version: 1.1

**Comment by l7ssha** [comment]
Somethings wrong with the implementation I think. Will take a look

**Comment by l7ssha** [comment]
Fixed in 94b227e
